src/UserRepository/PdoDatabaseFactory.php

```diff
@@ -59,12 +59,11 @@ public function __invoke(ContainerInterface $container) : PdoDatabase
             );
         }
 
+        $connection = new PDO($pdo['dsn'], $pdo['username'] ?? null, $pdo['password'] ?? null);
+        $connection->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_SILENT);
```

Comment:
I think this should be removed in a future version therefore a new issue report should be opened for this.

Reference:

* https://github.com/php/php-src/blob/ae944503c34d7c89433eaefe607fc0d631d6f9ac/UPGRADING#L424-L428
* https://wiki.php.net/rfc/pdo_default_errmode